content/about.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "About the Inclusive Naming Initiative"
+---
+
+The Inclusive Naming Initiative’s mission is to help companies and projects remove all harmful and unclear language of any kind and replace it with an agreed-upon set of neutral terms. The initiative’s goal is to define processes and tools to remove harmful language from projects. This includes creating a comprehensive list of terms with replacements, language evaluation frameworks and templates, and infrastructure to aid the transition. RedHat, IBM, LF Networking, and the Kubernetes Naming WG are all already on board.
```

Comment:
Can this be phrased in the positive?  'remove' and 'harmful' carry negative energy with them.  Sometimes that's needful and appropriate, but I often find there is great power in rephrasing in the positive.

Reply:
@xmulligan? 

Again, i don't think this is a blocker for merging this PR. We can iterate.

Reply:
Absolutely not a blocker.  Please do merge.